quartz/bootstrap-cli.mjs

```diff
@@ -151,20 +168,76 @@ yargs(hideBin(process.argv))
     console.log()
     intro(chalk.bgGreen.black(` Quartz v${version} `))
     const contentFolder = path.join(cwd, argv.directory)
-    const setupStrategy = exitIfCancel(
-      await select({
-        message: `Choose how to initialize the content in \`${contentFolder}\``,
-        options: [
-          { value: "new", label: "Empty Quartz" },
-          { value: "copy", label: "Copy an existing folder", hint: "overwrites `content`" },
-          {
-            value: "symlink",
-            label: "Symlink an existing folder",
-            hint: "don't select this unless you know what you are doing!",
-          },
-        ],
-      }),
-    )
+    let setupStrategy = argv.setup?.toLowerCase()
+    let linkResolutionStrategy = argv.links?.toLowerCase()
+    const inDirectory = argv["in-directory"]
+    let hasAllCmdArgs = false
+
+    // If all cmd arguments were provided, check if theyre valid
+    if (setupStrategy && linkResolutionStrategy) {
+      // If setup isn't, "new", --in-directory argument is required
+      if (setupStrategy !== "new") {
```

Comment:
unsure if there is a native way to better handle arguments that depend on each other natively in yargs, i didnt find a way so i manually implemented it for now